tutorials/robot-marbles-part-1/robot-marbles-part-1.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>Please have the output cells show the results so that the reader can follow the tutorial on GitHub first before cloning the repo and running the notebook locally. This applies to all output cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cadCAD-org/cadCAD/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Will do on the final commit :+1: